openmdao/visualization/n2_viewer/tests/gui_test_models/bug_arrow_N2_TEST.html

```diff
@@ -0,0 +1,7975 @@
+<!doctype html>
```

Comment:
Are you sure these html files should be added to the repo? I thought they were just temporary test outputs.

Reply:
You are right I thought I added them to the .gitignore file

Reply:
Both `cases.sql` and `openmdao/cases.sql` should also be removed.